### DISCUSSION TITLE
Adding to /cmi clear and /cmi give

### DISCUSSION THREAD

**Discussion by natr566** [Zrips/CMI#7015] [Enhancement]
My suggestion is some small additions to the two commands /cmi clear and /cmi give
Addition for /cmi clear the ability to use nbt like a custom model data like in the give command.
Addition for /cmi give the ability to use negative numbers to remove items instead of giving them.

**Comment by mrfloris** [comment]
negative give is perhaps this (for what you want)
`/cmi clear (playerName) (item(:amount)(;data)) (-s) (+clearType)`
